### ISSUE TITLE
Cannot import from CSV to postgresql

### ISSUE THREAD

**Issue by tdashworth** [shlinkio/shlink#1947]
### Shlink version

3.7.0

### PHP version

?

### How do you serve Shlink

Openswoole Docker image

### Database engine

PostgreSQL

### Database version

16.1 (Debian 16.1-1.pgdg120+1)

### Current behavior

After testing out with SQLite, I wanted to move over to postresql. I first exported the sqlite version to CSV (from the web app). Then, in docker-compose, I spun up a db container and linked the two with environment variables. Next, added a volume mount for the csv file and ran `docker exec -it shlink-shlink-1 shlink short-url:import csv` after answering the questions I get: 

```
Importing short URLs
====================


In ExceptionConverter.php line 87:
                                                                                                                            
  An exception occurred while executing a query: SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syn  
  tax for type bigint: ""                                                                                                   
  CONTEXT:  unnamed portal parameter $1 = ''                                                                                
                                                                                                                            

In Exception.php line 28:
                                                                                                    
  SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for type bigint: ""  
  CONTEXT:  unnamed portal parameter $1 = ''                                                        
                                                                                                    

In Statement.php line 130:
                                                                                                    
  SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for type bigint: ""  
  CONTEXT:  unnamed portal parameter $1 = ''                                                        
                                                                                                    
```

Docker Compose: 
```yml
version: '3.0'

services:
  shlink:
    image: shlinkio/shlink:stable
    restart: unless-stopped
    networks:
      - cloudflared_proxy
      - default
    ports:
      - 8080:8080
    volumes:
      - ./backup.csv:/backup.csv:ro
    environment:
      DEFAULT_DOMAIN: [redacted]
      IS_HTTPS_ENABLED: true
      GEOLITE_LICENSE_KEY: ${GEOLITE_LICENSE_KEY}
      INITIAL_API_KEY: ${INITIAL_API_KEY}
      DB_DRIVER: postgres
      DB_NAME: shlink
      DB_USER: postgres
      DB_PASSWORD: ${DB_PASSWORD}
      DB_HOST: db

  db:
    image: postgres
    restart: unless-stopped
    networks:
      - default
    environment:
      POSTGRES_PASSWORD: ${DB_PASSWORD}

networks:
  cloudflared_proxy:
    name: cloudflared_proxy
    external: true
  default:
    driver: bridge
```

### Expected behavior

The 15 or so links to be imported.

### How to reproduce

1. Step a standard docker instance. 
2. Add links
3. Export links to csv
4. Add postgresql db to docker instance
5. Try to import csv via cli

**Comment by tdashworth** [comment]
I just tired importing back into SQLite - it worked. 
I just tried importing into a new shlink (postresql) instance from the original (SQLite) instance - failed, same reason. 

It seems like postgresql is the issue. 

I also tested the postgresql instance by adding a link via the web app - it worked.

**Comment by acelaya** [comment]
Thanks!

There are some variations on how stuff is mapped into native database types. I guess Postgres is being more strict about some specific value.

I think I should be able to reproduce with all the details you provided.

**Comment by acelaya** [comment]
I have managed to reproduce it.

**Comment by acelaya** [comment]
This is now fixed. It will release with Shlink 3.7.1 before end of year.

**Comment by acelaya** [comment]
I have just released [v3.7.1](https://github.com/shlinkio/shlink/releases/tag/v3.7.1).

Docker images will be available once this build finishes: https://github.com/shlinkio/shlink/actions/runs/7240461338

**Comment by tdashworth** [comment]
Migration worked perfectly, thanks!
